src/engine.cairo

```diff
@@ -137,13 +137,18 @@ pub impl EngineImpl of EngineTrait {
                 }
 
                 if !self.cond_stack.branch_executing() && !flow::is_branching_opcode(opcode) {
-                    let res = Opcode::is_opcode_disabled(opcode, ref self);
-                    if res.is_err() {
-                        err = res.unwrap_err();
-                        break;
+                    if flow::is_pushing_opcode(opcode) {
+                        self.opcode_idx += 2;
```

Comment:
The push opcode wont always contain 2 bytes of data, it depends on the opcode.
I recommend looking at OP_DATA_X and OP_PUSHDATAX opcodes for more info

Reply:
To verify all the posibilities of how many bytes are gone be push I refactor the validations
I use this case as example (If it is wrong please let me know please)

0x01  0x4c 0x04 0x11223344

1 - The IF of the line 139 validates that the op_code is OP_DATA_1 or OP_DATA_75, this OP_CODES just push the next bytes into the array.
2 - The IF of the line 141 validates if  next_opcode is a pushing_opcode.
     - If the result is true: the engine will verify if the next_opcode(2) is disable and then update the index and continue.
     - If the result is false: the index will update an ommit the verification of the op that is going to be add to the stack.